src/eh1/delay.rs

```diff
@@ -89,55 +87,52 @@ impl Transaction {
         }
     }
 
-    /// Create a new delay_ns transaction
+    /// Crete a new delay_ns transaction
```

Comment:
Typo: `Crete` -> `Create`. There are other occurrences of "crete" in this file as well.